File: Resueltos_por_competidor/Juan_Martinez/StringTask.cpp
/*
 * Autor: Juan Martinez
 * Problema: String Tasks (118A)
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/problemset/problem/118/A
 * Difficulty: 1000
 */
#include <bits/stdc++.h>
#include <cctype>
using namespace std;

int main() {
	string s; cin >> s;
    queue<char> words;
    for(int i = 0; i < s.size(); i++) {
        s[i]  = tolower(s[i]);
        if(s[i] != 'a' && s[i] != 'o' && s[i] != 'y' && s[i] != 'e' && s[i] != 'u' && s[i] != 'i') words.push(s[i]);
    }
    stringstream ss;
    while(!words.empty()){
        ss<<'.';
        ss<<words.front();
        words.pop();
    }
    cout<<ss.str()<<endl;
}

File: Resueltos_por_competidor/Santiago_Ortiz/StringTask.java
import java.util.Scanner;

/*
 * Autor: Darkovski7   
 * Problema: String Task
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/118/A
 */

public class StringTask {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		StringBuilder resultado = new StringBuilder();
		
		String palabra;
		palabra = sc.next().toLowerCase();
		
		for (int i = 0; i < palabra.length(); i++) {
			char c = palabra.charAt(i);
			if(c != 'a' && c != 'e' && c != 'i' && c != 'o' && c != 'u' && c != 'y') {
				
				resultado.append('.').append(c);
			}
		}
		System.out.println(resultado.toString());
	}

}
